Levels/sandboxj.level

```diff
@@ -0,0 +1,248 @@
+Copyright (c) 2010 Edvin "Lego3" Linge
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+of the Software, and to permit persons to whom the Software is furnished to do
+so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Only lines beginning with % or # as explained below will be considered by the game.
+
+Each line beginning with a %-sign is a directive to use when loading the game,
+such as the name of the level and where to find the map file. Available commands:
+- Name: What the level should be called (within quotes)
+- MapFile: The name of the binary map file to load. First the Levels directory of the
+  original game will be searched, and then the Levels directory of CorsixTH.
+- LevelBriefing: The body of text shown before you start a custom scenario,
+  and as first information when you have just started a new level.
+  Everything between the citation marks will get included. If you use new lines:
+  One new line is simply discarded, while two new lines in a row creates
+  a new paragraph. So does the special expression // (legacy from Theme Hospital)
+- LevelDebriefing: Text shown in the fax when you win a level.
+
+Lines that begin with # defines all parameters for the level.
+By default there are no diseases and only some basic rooms are available.
+
+
+---------------------- General Information -------------------------
+%Name = "Sandbox"
+%MapFile = "sandboxj.map"
+
+Everything within citation marks will be the briefing/debriefing.
+Two new lines creates a new paragraph, but additional new lines in a row are discarded.
+%LevelBriefing = "Sandboxmap build build build"

```

Comment:
Just to verify, is this the only section where text has been changed? Or are these other parts of this file also changed?